src/Formatter/FormatterManager.php

```diff
@@ -212,9 +212,9 @@ public function getPlugin($instance_id) {
   protected function getPluginByName($name) {
     /* @var FormatterInterface $formatter */
     $formatter = $this->plugins->get($name);
-    $formatter->setConfiguration(array(
-      'resource' => $this->resource,
-    ));
+    if ($this->resource) {
+      $formatter->setResource($this->resource);
+    }
```

Comment:
This was a problem for tests. The resource plugin was stored and could not be changed for the subsequent test. This was not an issue during regular processing.